BitmovinPlayerSSAI/BitmovinPlayerSSAI/ContentView.swift

```diff
@@ -0,0 +1,105 @@
+import SwiftUI
+import BitmovinCollector
+import BitmovinPlayer
+import CoreCollector
+
+struct ContentView: View {
+    private let player: Player
+    private let playerViewConfig: PlayerViewConfig
+    private let ssaiAdProvider: SsaiInformationProvider = StaticSsaiInformationProvider()
+    private let bitmovinAdScheduler: BitmovinPlayerSsaiAdScheduler
+    private var sourceConfig: SourceConfig {
+        guard let streamUrl = URL(string: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8") else {
```

Comment:
It would be really cool if we create an asset with real ads in it. We should have enough content to do so.

Reply:
I synced with the team and we decided to leave it like this, due to the cost implications.